examples/oneview_server_hardware_facts.yml

```diff
@@ -50,6 +52,13 @@
 
     - debug: var=server_hardwares
 
+    - name: Gather facts about a Server Hardware by uri
+      oneview_server_hardware_facts:
+        config: "{{ config }}"
+        uri: "{{ server_hardware_uri }}"
+      delegate_to: localhost
+
+    - debug: var=server_hardwares
```

Comment:
can you paste the output of this play.? I want to see what output you are getting here.

Reply:
you will get the server_hardware details particular to URI given